docs/software/ml/pytorch.md

```diff
@@ -1,6 +1,207 @@
-[](){#ref-uenv-pytorch}
+[](){#ref-software-ml-pytorch}
 # PyTorch
 
+PyTorch is available both as a container with the [Container Engine (CE)][ref-container-engine] and a [uenv][ref-uenv] software stack. The best choice for your use case depends on the amount of control required over the lower level libraries.
+
+While NGC provides an optimized build of PyTorch with many dependencies included, uenv allows a more flexible choice of lower level libraries and represents a thinner layer over the host system. Both options can be customized - a container via a Dockerfile and a uenv (in advanced use cases) via its recipe and both, additionally, via Python virtual environments built on top. Due to the simplicity and reproducible performance, containers are generally the recommended default for most users.
+
+[](){#ref-ce-pytorch}
+## Running PyTorch with the Container Engine (recommended)
+
+Running PyTorch from a container ensures maximum portability, reproducibility, and ease of use across machines. This is achieved by 
+
+1. selecting an appropriate base image and customizing it in a Dockerfile
+2. define the container runtime environment in an EDF
+3. (optionally) extending by a virtual environment
+4. submitting jobs with CE in SLURM 
+
+These steps are illustrated in the [machine learning platform tutorials][ref-software-ml-tutorials] and the instructions detailed in the [podman build guide][ref-build-containers].
+
+!!! info "Preliminary steps"
+    Before proceeding with the next steps, make sure you have storage for podman configured as in the [build guide][ref-build-containers-configure-podman] and make sure to apply [recommended Lustre settings][ref-guides-storage-lustre] to every directory (e.g. `$SCRATCH/ce-images`) dedicated to container images before importing them with enroot. This is necessary to guarantee good filesystem performance.
+
+    ```bash
+    lfs setstripe -E 4M -c 1 -E 64M -c 4 -E -1 -c -1 -S 4M $SCRATCH/ce-images # (1)!
```

Comment:
same here:
I think it is not a good a idea to duplicate the command as you already linked it above. Also 64MB still seems a bit little for full striping, doesn't it?